spec/src/main/asciidoc/messaging-spec.adoc

```diff
@@ -3102,7 +3101,7 @@ send(Destination destination, Message message)
 
 ____
```

Comment:
Quote block is not required
![image](https://user-images.githubusercontent.com/16149023/81456641-ba0e6a80-91b0-11ea-9141-3a9fa42e842c.png)

Reply:
Fixed.